packages/statistic-link-dereference/README.md

```diff
@@ -0,0 +1,23 @@
+# Comunica Statistic Link Dereference
+
+[![npm version](https://badge.fury.io/js/%40comunica%2Fstatistic-link-dereference.svg)](https://www.npmjs.com/package/@comunica/statistic-link-dereference)
+
+A statistics tracker package for tracking link dereferences during query execution in Comunica. This class must be added to the context with the key defined in the class.
+
+This module is part of the [Comunica framework](https://github.com/comunica/comunica),
+
+[Click here if you just want to query with Comunica](https://comunica.dev/docs/query/).
+
+## Install
+
+```bash
+$ yarn add @comunica/statistic-link-dereference
+```
+
+## Adding to the context
+
+```javascript
+    let context = new ActionContext();
+    const statisticTracker = new StatisticLinkDereference();
+    context = context.set(statisticTracker.key, statisticTracker);
```

Comment:
It may be more helpful to add an example where a query engine is invoked and a context is passed.

We don't need an actual query though, something like `'...'` may be sufficient.